CUSTOMIZE.md

```diff
@@ -110,3 +110,27 @@ class Keybinding(object):
 1. Create a ```keybinding``` directory in the user directory
 2. Copy over one of the core keybindings, and customize to your liking.
 3. Set the ```default_keybindings``` setting in ```config.ini``` to the name of the keybinding file you created, without the ```.ini``` extension. For example, if you created ```keybinding/strange.ini```, you would set ```default_keybindings = strange``` in ```config.ini```
+
+#### Keybinding suggestions
+
+##### Jumping with digits
+
+As jumping to tasks is central to vit's operation, you might want to map each
```

Comment:
Most of the rest of the doc has `VIT` capitalized, can you make that the same here?